dashboard/test/ui/runner.rb

```diff
@@ -720,6 +720,7 @@ def cucumber_arguments_for_feature(options, test_run_string, max_reruns)
 end
 
 def to_percent(number, n_sig_digits)
+  return nil if number.nil?
```

Comment:
this check is necessary to avoid throwing an exception if `flakiness_for_test` returns `nil` for whatever reason. I don't think we want to block our ability to run the test on whether or not we have flakiness data